src/Workspaces/MSBuild/BuildHost/MSBuild/ProjectFile/ProjectFileLoader.cs

```diff
@@ -27,4 +29,18 @@ public async Task<ProjectFile> LoadProjectFileAsync(string path, ProjectBuildMan
 
         return this.CreateProjectFile(project, buildManager, log);
     }
+
+    public ProjectFile LoadProjectFile(string path, string projectContent, ProjectBuildManager buildManager)
```

Comment:
```suggestion
    public ProjectFile LoadProject(string path, string projectContent, ProjectBuildManager buildManager)
```

Just to match your naming elsewhere.